tests/integration/customer-import.js

```diff
@@ -134,22 +135,109 @@ describe('customer import module', function () {
       }]
     }
     customerImport.loadCustomerGroups()
+    .then(() => customerImport.importCustomer(customer))
     .then(() => {
-      customerImport.importCustomer(customer)
-      .then(() => {
-        const summary = JSON.parse(customerImport.summaryReport())
-        const actual = summary.errors.length
-        const expected = 0
+      const summary = JSON.parse(customerImport.summaryReport())
+      const actual = summary.errors.length
+      const expected = 0
 
-        expect(actual).to.equal(expected)
-        client.customers.where(`email="${customer.email}"`).fetch()
-        .then(({ body: { results: customers } }) => {
-          const actual = _.omit(customers[0].addresses[0], 'id')
-          const expected = customer.addresses[0]
+      expect(actual).to.equal(expected)
+      return client.customers.where(`email="${customer.email}"`).fetch()
+    })
+    .then(({ body: { results: customers } }) => {
+      const actual = _.omit(customers[0].addresses[0], 'id')
+      const expected = customer.addresses[0]
 
-          expect(actual).to.deep.equal(expected)
-          done()
-        })
+      expect(actual).to.deep.equal(expected)
+      done()
+    })
+    .catch(done)
+  })
+
+  it('should import a customer with a default shipping and billing address',
+  (done) => {
+
+    const customer = {
+      email: 'philipp.sporrer@commercetools.de',
+      addresses: [{
+        streetName: 'Ernst-Platz-Straße 45a',
+        postalCode: '80992',
+        city: 'München',
+        country: 'DE'
+      }]
```

Comment:
in this case you should rewrite history of this repository so that this commit doesn't get stored.